### ISSUE TITLE
Slow first launch for some GTK apps (again)

### ISSUE THREAD

**Issue by nwg-piotr** [swaywm/sway#6587]
- **Sway Version:**
  - 1.6.1
  - 1.6-4cdc4ac6
  - 1.6-8d0a84f3

- [Debug Log](https://gist.github.com/nwg-piotr/932e71d22eeb1d6e8afa438e29b760fb)

- **Configuration File:**
  - default, and also my customized, that used to work so far, including [this](https://github.com/swaywm/sway/wiki#gtk-applications-take-20-seconds-to-start) solution

- **Description:**
  -  After yesterday's Arch update it looks as if #5732 reappeared. First launch of e.g. mousepad, thunderbird, nwggrid takes ages. Having launched one of above once, I no longer have to wait while launching any GTK app.

**Comment by oscarpax** [comment]
FWIW, removing `xdg-desktop-portal-wlr` and `xdg-desktop-portal` did the trick for me. I did not have to remove `gnome-keyring`

**Comment by progandy** [comment]
This may be fixed in the xdg-desktop-portal git master ? https://github.com/flatpak/xdg-desktop-portal/issues/636

**Comment by nwg-piotr** [comment]
Let's see. I don't really need xdg-desktop-portal on my main machine.

**Comment by aragon999** [comment]
Thanks @progandy installing the current git version of `xdg-desktop-portal` fixed the issue for me.

**Comment by nwg-piotr** [comment]
> installing the current git version of `xdg-desktop-portal` fixed the

Very true. Closing.

**Comment by AzP** [comment]
My system has been working fine with the previously recommended fixes, but recently it started acting up again with the delays when starting Waybar and other GTK-dependent applications.

After much searching I've found that there seems to be a new issue in gtk-desktop-portal and perhaps gtk-desktop-portal-gnome, where the latter doesn't start properly if it is unable to find a running Mutter.

https://bugzilla.redhat.com/show_bug.cgi?id=2176759
https://gitlab.gnome.org/GNOME/xdg-desktop-portal-gnome/-/issues/74

Just mentioning it here, so others are able to find out about it. There seems to be fixes coming in.

I was able to work around the problem for now by just removing xdg-desktop-portal-gnome from my system.
